Core/clim-basic/text-formatting.lisp

```diff
@@ -31,11 +31,11 @@
 
 (defclass standard-page-layout ()
   ((%page-region :reader stream-page-region :writer (setf %page-region))
-   (margins :initarg :text-margins :accessor stream-text-margins :type margin-spec))
-  (:default-initargs :text-margins '(:left   (:absolute 0)
-                                     :top    (:absolute 0)
-                                     :right  (:relative 0)
-                                     :bottom (:relative 0))))
+   (margins :initform '(:left   (:absolute 0)
```

Comment:
I have to admit that I don't understand how this fix addresses the problem. But, I've tested it and it does. So I can't really complain about that.